src/proxy/console.js

```diff
@@ -0,0 +1,33 @@
+const methods = [
+    'error',
+    'info',
+    'warn'
+];
+const listeners = [];
+
+methods.forEach(function(methodKey) {
```

Comment:
I'd add a todo comment here to get this directly from `Glimpse.Browser.Agent` in the future.